File: C/for_loops & while/if_else_if/weekdays.c
// week days :
// 1:sund 
// 2:monday

// otherwise:input is not matched 

#include<stdio.h>

int main () {
    int num;
    printf("Enter the number value");
    scanf("%d",&num);
    if(num == 1){
        printf("Sunday");
    } else if (num == 2) {
      printf("Sunday");
    } else {
      printf("Input is not matched ");
    }
    return 0;
}

File: src/Java/If else/Weekdays.java
//13)
//        number (WEEK DAYS)
//        1 =>sunday
//        2 =>monday
//        3 =>tuesday

import java.util.Scanner;

public class Weekdays {
    public static void main(String[] args) {
        int day;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the day number: ");
        day = sc.nextInt();
        if (day == 1) {
            System.out.println( "Monday");
        } else if (day == 2) {
            System.out.println( "Tuesday");
        } else if (day == 3) {
            System.out.println( "Wednesday");
        } else {
            System.out.println( "Invalid Day");
        }
    }
}